plugins/vpp/l3plugin/data_resync.go

```diff
@@ -17,10 +17,14 @@ package l3plugin
 import (
 	"fmt"
 
+	"strings"
+
 	"github.com/ligato/vpp-agent/plugins/vpp/l3plugin/vppcalls"
 	"github.com/ligato/vpp-agent/plugins/vpp/model/l3"
 )
 
+const ipv6LinkLocalPrefix = "fe80:"
```

Comment:
https://godoc.org/net#IP.IsLinkLocalUnicast

Reply:
I know about it, but resync works with the string representation of IP addresses (NB config, dumped data, ...), I see no point converting it to the net.IP just because of the check, which - if you have a look at the implementation - does basically the same thing